.gitignore

```diff
@@ -5,10 +5,15 @@ tags/
 *.pyc
 *.bak
 BuildConfig.conf
+/Conf
+/.vscode
+SharedCrypto_extdep/
```

Comment:
this shouldn't be needed as wildcard should cover

Reply:
Done